public/app/plugins/datasource/cloudwatch/datasource.js

```diff
@@ -191,7 +185,20 @@ function (angular, _) {
           });
         }
 
-        return this.getDimensionValues(region, namespace, metricName, dimensions);
+        return this.getDimensionValues(region, namespace, metricName, dimensions).then(function(result) {
+          return _.map(result, function(dimensions) {
+            var values = _.chain(dimensions)
+            .sortBy(function(dimension) {
+              return dimension.Name;
+            })
+            .map(function(dimension) {
+              return dimension.Name + '=' + dimension.Value;
```

Comment:
Ah yes, I want to separate these if possible...

For example, if dimension name/value pairs like this.
CacheClusterId=cluster-001
CacheClusterId=cluster-001,CacheNodeId=0001
CacheClusterId=cluster-001,CacheNodeId=0002
CacheClusterId=cluster-001,CacheNodeId=0003
CacheClusterId=cluster-002
CacheClusterId=cluster-002,CacheNodeId=0001
CacheClusterId=cluster-002,CacheNodeId=0002

And if separete template variable for CacheClusterId and CacheNodeId.
The CacheClusterId variable has "001", "002".
The CacheNodeId variable has "001", "002", "003". (Because, cluster-001 has 003 node.)

From these templating variable, we can't reproduce original dimension name/value pairs.
If using these templating variable for Panel repeating, the Grafana will make "CacheClusterId=cluster-002,CacheNodeId=0003" graph, it is blank!